docs/changelog.md

```diff
@@ -44,6 +44,7 @@
 * Fix #109: The player now correctly loses 10 ore if he chooses to pay protection money to Bloodwyn later.
 * Fix #111: The player now correctly loses 10 ore if he chooses to pay protection money to Jackal.
 * Fix #112: The player now correctly loses 10 ore if he chooses to pay protection money to Jackal later.
+* Fix #124: The gate guard now correctly closes the castle gate of the Old Camp in chapter 4.
```

Comment:
This description depends on #163 now. Should we rephrase it?

Reply:
*Gate guard of the castle now has the correct routine and stands at the gate.*

Reply:
Thanks, I added a bit more detail, see 3b8d863.